src/response.js

```diff
@@ -124,6 +124,25 @@ export default class Response extends Body {
 		return response;
 	}
 
+	static json(data = undefined, init = {}) {
```

Comment:
I think that using `init = {}` and assuming `init` is an object further down in the function will make the following code crash:

```js
Response.json({ hej: 1 }, null)
```

But in Chrome that works:

![Screenshot 2022-11-09 at 09 22 54](https://user-images.githubusercontent.com/189580/200776898-9e09bb65-5d87-4fbf-bb84-4005ccf7bd24.png)

Reply:
node-fetch doesn't seem to do much argument validation, I followed other areas of the code.

For example when constructing a new response:
```js
new Response('', null)
```

throws an error when it shouldn't

```js
new Response('', 3)
```
works when it shouldn't (same with booleans, etc)